lib/cws/server.ts

```diff
@@ -26,6 +26,8 @@ export class WebSocketServer extends eventEmitter() {
     if (configs.path && configs.path[0] !== '/') {
       configs.path = `/${configs.path}`;
     }
+    
+    this._upgradeCallback = noop;
```

Comment:
Can you rename variables to just `upgradeCallback` and move at the top with private tag (it is TypeScript not a JS so it has slightly different approach in creating variables) for reference check out `upgradeListener` variable.